src/components/Tooltip/Tooltip.spec.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import { fireEvent, render, waitForDomChange } from '@testing-library/react';
+
+import Tooltip from './Tooltip';
+
+const getProps = (props = {}) => ({
+  className: props.className,
+  disableFocusListener: props.disableFocusListener,
+  disableMouseListener: props.disableMouseListener,
+  enterDelay: props.enterDelay !== undefined ? props.enterDelay : 0,
+  leaveDelay: props.leaveDelay !== undefined ? props.leaveDelay : 0,
+  onBlur: jest.fn(),
+  onClose: jest.fn(),
+  onFocus: jest.fn(),
+  onMouseEnter: jest.fn(),
+  onMouseLeave: jest.fn(),
+  onOpen: jest.fn(),
+  style: props.style,
+  testId: 'tip',
+  text: 'I am the tooltip'
+});
+
+const getTooltip = props => (
```

Comment:
I know this is kinda dumb but what's your opinion about `getTooltip` instead of `renderTooltip`? 😂 

_Ps: This change won't block the PR, just asking to see your opinion haha_

Reply:
i'm down with it!

Reply:
makes sense considering it returns JSX

Reply:
updated in https://github.com/arturbien/React95/commit/82e7600b527fd3f9a89548b1231816ec50a3680c